activiti-services/activiti-services-events/src/main/java/org/activiti/services/events/MessageProducerActivitiEventListener.java

```diff
@@ -24,10 +29,34 @@ public MessageProducerActivitiEventListener(ProcessEngineChannels producer,
 
     @Override
     public void onEvent(ActivitiEvent event) {
+        CommandContext currentCommandContext = Context.getCommandContext();
```

Comment:
I added a unit test.
https://github.com/Activiti/Activiti/pull/1424/commits

But, I couldn't mock  Context.getCommandContext() because PowerMock is not compatible with Spring Boot. I used old activiti unit test style. 
https://github.com/powermock/powermock/issues/695